gobblin-runtime/src/test/java/org/apache/gobblin/runtime/spec_store/MysqlBaseSpecStoreTest.java

```diff
@@ -89,6 +93,22 @@ public void setUp() throws Exception {
         .withVersion("Test version 2")
         .withSpecExecutor(MockedSpecExecutor.createDummySpecExecutor(new URI("execB")))
         .build();
+
+    flowSpec = new FlowSpec.Builder(this.uri1) .withConfig(ConfigBuilder.create()
+            .addPrimitive("properties.user.to.proxy", "value1")
```

Comment:
given [how things work](https://github.com/apache/gobblin/blob/d9f83e638b6eb34fa85b0317eee31cf39c1e8546/gobblin-runtime/src/main/java/org/apache/gobblin/runtime/spec_store/MysqlSpecStore.java#L82) in `completeInsertPreparedStatement`, I don't believe you ought to include the `properties.` prefix.  AFAIR that is only needed when passing prop updates via carbon